<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LSH</class>
 <widget class="QWidget" name="LSH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSH</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="LSH_tune_btn">
     <property name="text">
      <string>Tune</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="LSH_gb">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>LSH grid file:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="LSH_load_btn">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="LSH_apply_btn">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLineEdit" name="LSH_file_le"/>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="LSH_enable_cb">
        <property name="text">
         <string>Enable use of lens shading grid</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="LSH_info_l">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>LSH_load_btn</tabstop>
  <tabstop>LSH_apply_btn</tabstop>
  <tabstop>LSH_file_le</tabstop>
  <tabstop>LSH_enable_cb</tabstop>
  <tabstop>LSH_tune_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
